.github/ISSUE_TEMPLATE/support.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Support Request
+about: Support request or question relating to Kubernetes federation v2
+
+---
+
+<!-- 
+STOP -- PLEASE READ!
+
+GitHub is not the right place for support requests.
+
+If you're looking for help, check [Stack Overflow](https://stackoverflow.com/questions/tagged/kubernetes) and the [troubleshooting guide](https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/).
```

Comment:
If we want to direct users to stack overflow, I think we'll need to add some questions and answers with an appropriate tag (e,g. fedv2 or kubefedv2) and link to a fedv2-specific troubleshooting guide.